packages/test/test-service-load/src/runner.ts

```diff
@@ -152,23 +152,34 @@ function* factoryPermutations<T extends IDocumentServiceFactory>(create: () => T
 		// Certain behavior (like driver caches) are per factory instance, and by reusing it we hit those code paths
 		// At the same time we want to test newly created factory.
 		let documentServiceFactory: T = factoryReused;
-		let headers: IRequestHeader = {};
+		let headers: IRequestHeader = { deltaConnection: "none" };
```

Comment:
this doesn't look valid, shouldn't it be 
```suggestion
		let headers: IRequestHeader = { [LoaderHeader.loadMode]: {deltaConnection: "none" }};
```